subgraphs/convex-finance/src/Prices/index.ts

```diff
@@ -15,11 +17,19 @@ import { getPriceUsdc as getPriceUsdcSushiswap } from "./routers/SushiSwapRouter
 import { getTokenPriceFromSushiSwap } from "./calculations/CalculationsSushiswap";
 import { getTokenPriceFromCalculationCurve } from "./calculations/CalculationsCurve";
 
-export function getUsdPricePerToken(tokenAddr: Address): CustomPriceType {
+export function getUsdPricePerToken(
+  tokenAddr: Address,
+  block: ethereum.Block
+): CustomPriceType {
   // Check if tokenAddr is a NULL Address
-  if (tokenAddr.toHex() == constants.ZERO_ADDRESS_STRING) {
+  if (tokenAddr.toHex() == constants.ZERO_ADDRESS_STRING)
+    return new CustomPriceType();
+
+  if (
+    tokenAddr.equals(constants.CRV_TRI_CRYPTO_ADDRESS) &&
+    block.number.lt(BigInt.fromI32(12936339))

```

Comment:
You should add a comment here and link to this PR

Reply:
Added